### PR TITLE
Copy update on /download/server/arm

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Updates links based on [copy doc](https://docs.google.com/document/d/1whVU3DvB9j5k6Ed3UvLZCEN6BU-lZcow4ZefqKQqokg/edit)

## QA

- Go to download/server/arm and check the links are updated

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10158

### CODE REVIEW

File: templates/download/server/arm.html [hunk 1]
```diff
@@ -48,9 +48,9 @@ <h2 class="p-heading--5">Ubuntu Server</h2>
             </a>
           {% endif %}
           <div class="p-section--shallow">
-            <a href="/download/alternative-downloads">Alternative and previous releases&nbsp;&rsaquo;</a>
+            <a href="https://cdimage.ubuntu.com/releases">Alternative and previous releases&nbsp;&rsaquo;</a>
             &nbsp;&nbsp;
-            <a href="https://wiki.ubuntu.com/ARM/Server/Install">How to install&nbsp;&rsaquo;</a>
+            <a href="https://ubuntu.com/tutorials/install-ubuntu-server#1-overview">How to install&nbsp;&rsaquo;</a>
```
Comment (minkyngkm) [inline templates/download/server/arm.html:53]:
```suggestion
            <a href="/tutorials/install-ubuntu-server#1-overview">How to install&nbsp;&rsaquo;</a>
```

**Review by minkyngkm — APPROVED** [review]
Just left a comment for a tiny thing. LGTM!

**Comment by minkyngkm** [comment]
oh by the way its not from this PR but on the page linked from "how to install" https://ubuntu.com/tutorials/install-ubuntu-server#1-overview, steps look weird..
image

**Comment by lorumic** [comment]
> oh by the way its not from this PR but on the page linked from "how to install" https://ubuntu.com/tutorials/install-ubuntu-server#1-overview, steps look weird..

Seems to be the same issue addressed by [this](https://github.com/canonical/canonical.com/pull/1223) PR.

**Pull request merged**